Source/Scene/Expression.js

```diff
@@ -56,8 +56,11 @@ define([
 
     });
 
-    Expression.prototype.evaluate = function(feature) {
-        return this._runtimeAst.evaluate(feature);
+    Expression.prototype.evaluate = function(feature, result) {
```

Comment:
Would this be called when we are expecting colors? The problem with that is that there are still expressions that could involve Colors and Color operations without evaluating to colors, for example `color(...) * 3 === color(...)`.